scapy/layers/tls/handshake.py

```diff
@@ -349,7 +419,7 @@ def dispatch_hook(cls, _pkt=None, *args, **kargs):
     def post_build(self, p, pay):
         if self.random_bytes is None:
             p = p[:10] + randstring(28) + p[10 + 28:]
-        return super(TLSClientHello, self).post_build(p, pay)
+        return super(TLSServerHello, self).post_build(p, pay)
```

Comment:
Does this make sense?

Reply:
I've made this change because I had an issue with Codacy with the new TLS13ServerHello's post_build() method, which was relatively similar to the TLSServerHello's post_build() method. 

The Codacy error report said : "Bad first argument 'TLSClientHello' given to super()" and suggest to "Enforces the super() first argument to be the type of the current class."

So, I applied the changes on both TLS13ServerHello and TLSServerHello post_build() method (and changed the parents class, because it's the generic post_build method from _TLSHandshake that need to be called). And the same for the tls_session_update method.